dashboard/src/components/SideMenu/SendFeedback.tsx

```diff
@@ -68,14 +74,14 @@ const SendFeedback = (
           </AlertDialogHeader>
           <AlertDialogFooter>
             <AlertDialogAction asChild>
-              <a onClick={onGitHubClick}>
+              <Link onClick={onGitHubClick}>
```

Comment:
instead of `getGithubBody` to be afunction, it will be just an useMemo based on that state

Reply:
also, instead of window I think we should probably use `TanstackRouter`, it probably has a way to get the path 
take a look at

https://tanstack.com/router/latest/docs/framework/react/api/router/useLocationHook

Reply:
Using useLocation Hook from TanstackRouter, we access only the pathname. The address BASE_URL/tree?origin return /tree, but we need the complete url. Why would I use this hook if the window.location.href return the complete url?